tests/verify_membackend.erl

```diff
@@ -152,7 +152,10 @@ check_put_delete(Node) ->
     Mem = get_used_space(Pid, Node),
 
     %% this is meh, but the value isn't always the same length.
-    case (Mem == MemBaseline - 1142) orelse 
+    %% It seems to be the size of a Riak Object
+    case (Mem == MemBaseline - 1146) orelse
```

Comment:
I wonder if this test would be less brittle if it just set some kind of upper threshold like "Baseline - 1150" and made sure that Mem > Threshold or something like that.  Maybe there's some reason the test isn't already like that?